.github/workflows/install.yml

```
@@ -120,12 +127,24 @@ jobs:
         with:
           main-branch-name: ${{ steps.get-branch.outputs.MAIN_BRANCH }}
 
+      - name: Prepare feature deployment
+        if: inputs.DEPLOY_FEATURE == 'true'
+        uses: ./.github/actions/feature-checkout
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          slack_webhook_url: ${{ secrets.SLACK_BUILD_ISSUES_REPORTING_WEBHOOK_URL }}
+
+      - name: Prepare pre-release
+        if: inputs.PRE_RELEASE == 'true'
```

actionlint (1.7.4)

property "pre_release" is not defined in object type {main_branch: string; run_merge_queue: bool}

(expression)